### PR TITLE
Update CLI docs to mention telemetry

### PR DESCRIPTION
**Pull request by BeFunes**
# Description
We are introducing opt-telemetry into the CLI. 
Telemetry behaviour and instructions for enabling and disabling should be documented in [this page](https://circleci.com/docs/local-cli/)

# Reasons
Ticket --> [DEVEX-1238](https://circleci.atlassian.net/browse/DEVEX-1238)
(Blocked by [DEVEX-898](https://circleci.atlassian.net/browse/DEVEX-898)) 

# Content Checklist
Please follow our style when contributing to CircleCI docs. Our style guide is here: [https://circleci.com/docs/style/style-guide-overview](https://circleci.com/docs/style/style-guide-overview).

Please take a moment to check through the following items when submitting your PR (this is just a guide so will not be relevant for all PRs) 😸:

- [x] Break up walls of text by adding paragraph breaks.
- [x] Consider if the content could benefit from more structure, such as lists or tables, to make it easier to consume.
- [x] Keep the title between 20 and 70 characters.
- [x] Consider whether the content would benefit from more subsections (h2-h6 headings) to make it easier to consume.
- [x] Check all headings h1-h6 are in sentence case (only first letter is capitalized).
- [x] Is there a "Next steps" section at the end of the page giving the reader a clear path to what to read next?
- [x] Include relevant backlinks to other CircleCI docs/pages.

‼️ **This PR should not be merged before [DEVEX-1238](https://circleci.atlassian.net/browse/DEVEX-1238) is shipped** ⚠️

[DEVEX-1238]: https://circleci.atlassian.net/browse/DEVEX-1238?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DEVEX-898]: https://circleci.atlassian.net/browse/DEVEX-898?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: jekyll/_cci2/local-cli.md [hunk 1]
```diff
@@ -150,6 +150,19 @@ circleci setup
 
 The set up process will prompt you for configuration settings. If you are using the CLI with CircleCI cloud, use the default CircleCI host. If you are using CircleCI server, change the value to your installation address (for example, circleci.your-org.com).
 
+## Telemetry
+{: #telemetry }
+
+The CircleCI CLI includes a telemetry feature that collects basic errors and feature usage data in order to help us improve the experience for everyone.
+
+Telemetry works on an opt-in basis: when running a command for the first time, you will be asked for consent to enable telemetry. Telemetry is disabled by default for non-interactive terminals, ensuring that scripts that leverage the CLI run smoothly.
```
Comment (rosieyohannan) [inline jekyll/_cci2/local-cli.md:158]:
```suggestion
Telemetry works on an opt-in basis. When running a command for the first time, you will be asked for consent to enable telemetry. Telemetry is disabled by default for non-interactive terminals, ensuring that scripts that leverage the CLI run smoothly.
```

File: jekyll/_cci2/local-cli.md [hunk 2]
```diff
@@ -150,6 +150,19 @@ circleci setup
...
+
+You can disable or enable telemetry any time in one of the following ways:
+
+* Run the commands `circleci telemetry enable` or `circleci telemetry disable`
```
Comment (rosieyohannan) [inline jekyll/_cci2/local-cli.md:162]:
```suggestion
* Run one of the following commands: `circleci telemetry enable` or `circleci telemetry disable`
```

File: jekyll/_cci2/local-cli.md [hunk 3]
```diff
@@ -150,6 +150,19 @@ circleci setup
...
+
+* Set the `CIRCLECI_CLI_TELEMETRY_OPTOUT` environment variable to `1` or `true` to disable it
```
Comment (rosieyohannan) [inline jekyll/_cci2/local-cli.md:164]:
```suggestion
* To disable telemetry, set the `CIRCLECI_CLI_TELEMETRY_OPTOUT` environment variable to `1` or `true`
```

**Review by rosieyohannan — COMMENTED** [review]
Thank you @BeFunes!

**Comment by BeFunes** [comment]
@rosieyohannan made all the changes suggested in comments, thank you!! 

I expect that we will release the feature early next week. Should I give you a heads up then so the PR is merged, or should I just do it myself (if it's approved by then)?

**Review by rosieyohannan — APPROVED** [review]

**Comment by rosieyohannan** [comment]
@BeFunes You can go ahead and merge this whenever you're ready :-D

**Pull request merged**